### ISSUE TITLE
NER never recognizes any entities. Is the NER model file provided?

### ISSUE THREAD

**Issue by NSchrading** [honnibal/spaCy#52]
Trying to run the given NER test case, I get no entities in the list. Additionally, if I try making up a test case I always get 0 entities in the returned list. I have downloaded the latest files using
```
python -m spacy.en.download
```
and am using v0.82 on Python 3.4

```python
def test_simple_types():
    tokens = nlp(u'Mr. Best flew to New York on Saturday morning.')
    ents = list(tokens.ents)
    assert ents[0].start == 1
    assert ents[0].end == 2
    assert ents[0].label_ == 'PERSON'
    assert ents[1].start == 4
    assert ents[1].end == 6
    assert ents[1].label_ == 'GPE'
    assert ents[2].start == 7
    assert ents[2].end == 8
    assert ents[2].label_ == 'DATE'
    assert ents[3].start == 8
    assert ents[3].end == 9
    assert ents[3].label_ == 'TIME'
```
```
assert ents[0].start == 1
IndexError: list index out of range
```

**Comment by honnibal** [comment]
Hmm.

spacy.en.download should be downloading this file: https://s3-us-west-1.amazonaws.com/media.spacynlp.com/en_data_all-0.8.tgz , and the file should have a directory "ner".

Both seem true to me, and the test passed for me. Can you check that the data is ending up in the right place? It should be installed into spacy/en/data . You can find the directory of the spacy package with

    python -c "import os.path; import spacy.en; print(os.path.abspath(spacy.en.__file__))"

**Comment by cjeiler** [comment]
I'm seeing the same issue, I checked the spacy package directory to ensure the ner folder contained the model file (it did) and ran the test above to no avail. I'm always getting an empty list of entities and an assertion error for each of the tests. FWIW I'm running 2.7 on Ubuntu

**Comment by honnibal** [comment]
This should be fixed in v0.83, with new data pushed.

Run:

    $ pip install --upgrade spacy
    $ python -m spacy.en.download all

I'm still not 100% sure how I uploaded the broken model. Thanks for your patience.

**Comment by NSchrading** [comment]
Looks like it's working now. Thanks.
